mindsdb/integrations/handlers/rag_handler/settings.py

```diff
@@ -175,6 +175,7 @@ def load_vector_store_client(
                 folder_path=self.config.persist_directory,
                 embeddings=self.config.embeddings_model,
                 index_name=self.config.collection_name,
+                allow_dangerous_deserialization=True
```

Comment:
Without this it fails with FAISS, seems to be a new parameter added in later versions of langchain for added security as FAISS is stored as a binary which gets deserialized on loading